go/core/servers.go

```diff
@@ -70,7 +70,7 @@ func startDevServer(addr string) error {
 	// Don't use "localhost" here. That only binds the IPv4 address, and the genkit tool
```

Comment:
This comment needs fixing.
So "localhost" doesn't work, but "127.0.0.1" does?
(maybe take a look at the PR that added this comment)

Reply:
@apascal07 do you remember?

Reply:
I imagine it'll still only bind the IPv4 since it's an IPv4 address. Have you tried testing E2E?

Reply:
you mean with `genkit start`? yes, works fine.

Reply:
If the dev UI works then I think the original comment can be removed.

Reply:
removed